src/timeout.ts

```diff
@@ -180,20 +180,14 @@ export abstract class TimeoutContext {
 
   abstract get maxTimeMS(): number | null;
 
-  abstract get serverSelectionTimeout(): Timeout | null;
+  abstract get timeoutForServerSelection(): Timeout | null;
```

Comment:
half of our timeout methods were `timeoutFor<x>` and the other half were `<x>Timeout`.  I've aligned them all to `timeoutFor<x>`.